api/opensearch_dashboards.d.ts

```diff
@@ -0,0 +1,272 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/// <reference types="node" />
+
+import {
+    ClientOptions,
+    ConnectionPool,
+    Serializer,
+    Transport,
+    errors,
+    RequestEvent,
+    ResurrectEvent,
+    ApiError
+  } from '../index'
+  import Helpers from '../lib/Helpers'
+  import {
+    ApiResponse,
+    TransportRequestPromise,
+    TransportRequestParams,
+    TransportRequestOptions
+  } from '../lib/Transport'
+  import * as T from './types'
+  
+  /**
+    * We are still working on this type, it will arrive soon.
+    * If it's critical for you, please open an issue.
+    * https://github.com/elastic/elasticsearch-js
```

Comment:
should we modify this?

Reply:
thanks for catching this. I will modify it.